### PR TITLE
customize datafinder url

### PR DESCRIPTION
**Pull request by labkey-matthewb**

### CODE REVIEW

File: src/org/labkey/immport/ImmPortModule.java [hunk 1]
```diff
@@ -28,21 +29,28 @@
...
 
 import java.lang.reflect.InvocationTargetException;
+import java.net.URISyntaxException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.Set;
 
+import static com.rometools.utils.Strings.isBlank;
```
Comment (labkey-adam) [inline src/org/labkey/immport/ImmPortModule.java]: Wrong import, I suspect

File: src/org/labkey/immport/ImmPortModule.java [hunk 2]
```diff
@@ -143,4 +168,26 @@ public void doStartup(ModuleContext moduleContext)
     {
         return Collections.singleton("immport");
     }
+
+    public static URLHelper getDataFinderURL(Container c, User user)
+    {
+        URLHelper dataFinder = null;
+        ImmPortModule m = (ImmPortModule)ModuleLoader.getInstance().getModule("immport");
```
Comment (labkey-adam) [inline src/org/labkey/immport/ImmPortModule.java]: "immport" -> NAME?

File: src/org/labkey/immport/ImmPortModule.java [hunk 3]
```diff
@@ -91,8 +99,25 @@ protected void init()
         ModuleProperty proxyTarget = new ModuleProperty(this, "proxyTargetUri", ModuleProperty.InputType.text, "target for RApi proxy servlet (/_rapi/)", "target uri", false);
         proxyTarget.setEditPermissions(Arrays.asList(AdminOperationsPermission.class));
         addModuleProperty(proxyTarget);
+
+        ModuleProperty studyFinder = new ModuleProperty(this, "dataFinderUri", ModuleProperty.InputType.text, "URL for data finder, default /Studies/project-begin.view", "datafinder url", true)
```
Comment (labkey-adam) [inline src/org/labkey/immport/ImmPortModule.java]: Constant for "dataFinderUri"?

**Review by labkey-adam — CHANGES_REQUESTED** [review]
Change import. Other comments optional.

File: src/org/labkey/immport/ImmPortModule.java (hunk 1 above)
Comment (labkey-matthewb) [reply src/org/labkey/immport/ImmPortModule.java]: thank you

**Review by labkey-adam — APPROVED** [review]

**Pull request merged**